### PR TITLE
Refactor to Create React App boilerplate.

### PR DESCRIPTION
**Pull request by jpetto**
Big time changes, but will allow easy component re-use and get us on a more "standard" React path. Open to any and all suggestions here.

**tl;dr**

Based off [Create React App](https://github.com/facebookincubator/create-react-app). Local development happens in `local-dev` folder. After changes are finished, run a `build` command to package all the files, then a `finalize` command to move the build into the `gh-pages` root. Make the commit and push to get changes live. See `local-dev/README.md` for more detail.

Next steps would be determining `alerts/*.yml` format and building out basic service detail view.

A demo repo (with working gh-pages branch) is [here](https://github.com/jpetto/test.status.mozmar.org), with the running app [here](http://mozmar.equalparts.io/).

### CODE REVIEW

**Comment by glogiotatidis** [comment]
I suggest that we make use of the new github feature to [host github pages from `/docs` folder](https://github.com/blog/2228-simpler-github-pages-publishing) from the master branch. 

This change would:
- remove  `gh-pages` branch
- allow development of the app would still happen in the `local-dev` folder as suggested in this PR
- save final files into `/docs` in `master` branch instead of `gh-pages` root. 

Wins:
- Both python app and react app live in the same branch (but different directories) which ends the inconvenience of having to switch branches and make two pull requests when developing features that though both backend and frontend.
-  Reduces clutter in after  `finalize` command by moving all production files to `/docs`

Other than this looks good! Thanks @jpetto 🍰

**Comment by jpetto** [comment]
Oooh, I had not heard of this new feature. Good suggestion! Will close this PR and open a new one against the master branch.
